ui/components/multichain-accounts/multichain-account-cell/multichain-account-cell.tsx

```diff
@@ -100,7 +108,11 @@ export const MultichainAccountCell = ({
           borderColor={BorderColor.transparent}
           borderRadius={BorderRadius.XL}
         >
-          <PreferredAvatar address={seedAddressIcon} />
+          <ConnectedStatus
+            address={seedAddressIcon}
+            isActive={connectionStatus === STATUS_CONNECTED}
```

Comment:
Logic for active account is it should be connected and is the most recent one. I mean if more than one account is connected. How ill we get the active state?